src/shared/utils/exportData/getReportName.ts

```diff
@@ -12,15 +12,15 @@ export const getReportZipName = (name: ZipFile, suffix: string) =>
   `${name}-responses-${new Date().toDateString()}${suffix}.zip`;
 
 export const getStabilityTrackerCsvName = (
-  id: string,
+  item: DecryptedAnswerData,
   phaseType: DecryptedStabilityTrackerAnswerObject['phaseType'],
-) => `${id}_${phaseType}.csv`;
+) => `${item.targetSecretId}_${item.id}_${phaseType}.csv`;
```

Comment:
Indeed, your assessment is correct. I ran into this same issue initially (trouble telling where one UUID starts, and another begins). In fact without knowing the format beforehand, you don't even know what each UUID represents.

However, I wouldn't make this change as part of this ticket because there are stakeholders who are relying on the format of these file names so each change has to be carefully considered. This is something we could propose for the future

Reply:
Fair enough. I'll create a ticket!

Reply:
Based on the comments on [M2-6990](https://mindlogger.atlassian.net/browse/M2-6990) I'll resolve this thread